guide/src/main/asciidoc/problems/missingScope.adoc

```diff
@@ -13,7 +15,7 @@ Authorization: Bearer ds4d2f13sdds2q13qxcgbdf245
 ```
 HTTP/1.1 403 Forbidden
 Content-Type: application/problem+json
-
+WWW-Authenticate: Bearer realm="example", error="insufficient_scope",error_description="Missing scope to perform request"
```

Comment:
In https://github.com/belgif/rest-problem-java/blob/c36e0c9ab0e410b8fd2846d0fb7bfe520a645af0/belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/MissingScopeProblem.java#L106 we don't have an error_description. Maybe I should add one?